torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -824,6 +824,15 @@ class FullyShardedDataParallel(nn.Module):
         same FSDP unit. If enhanced shared parameter support is needed for your
         use case, please ping https://github.com/pytorch/pytorch/issues/77724
 
+    .. note:
+        Attempting to run the forward pass of a submodule that is contained in an
+        FSDP unit is not supported and will result in errors. This is because the
```

Comment:
nit: I think maybe we should converge to just saying FSDP instance (where instance is the technical OOP term). 
```suggestion
        FSDP instance is not supported and will result in errors. This is because the
```